variables.tf

```diff
@@ -200,3 +241,22 @@ variable "zone_dns_name" {
     error_message = "The DNS name cannot end with '.local'."
   }
 }
+
+# TODO
+
+variable "aws_region" {
+  description = "AWS region to deploy resources into"
+  type        = string
+}
+
+variable "azs" {
+  description = "Availability zones to use in AWS region"
```

Comment:
This doesn't seem to be used anywhere

Reply:
It is used in the tfvars file.

aws_region = "us-east-1"

azs = ["us-east-1a", "us-east-1b", "us-east-1c"]

tags = {
  Name = "GraphDB"
  Environment = "Production"
}